src/main/java/net/dv8tion/jda/client/entities/AuthorizedApplication.java

```diff
@@ -0,0 +1,95 @@
+/*
+ *     Copyright 2015-2017 Austin Keener & Michael Ritter
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.client.entities;
+
+import java.util.List;
+import net.dv8tion.jda.core.JDA;
+import net.dv8tion.jda.core.entities.ISnowflake;
+import net.dv8tion.jda.core.requests.RestAction;
+
+/**
+ * Represents a Discord Application from the point of view of a client havong authorized it.
```

Comment:
havong -> having.

What is the difference between an Application and an AuthorizedApplication (I know, but add a line in the class description).